.github/workflows/build.yml

```diff
@@ -250,26 +250,11 @@ jobs:
           cd python/railjson_generator
           poetry install
 
-      - name: Flake8
-        run: |
-          cd python/railjson_generator
-          poetry run pflake8 --config ./pyproject.toml --output-file flake8.xml --format junit-xml
-
-      - name: Publish flake8 report
-        uses: mikepenz/action-junit-report@v5
-        if: failure()
-        with:
-          report_paths: flake8.xml
-
-      - name: Black
-        run: |
-          cd python/railjson_generator
-          poetry run black . --check
-
-      - name: Isort
+      - name: Ruff
         run: |
           cd python/railjson_generator
-          poetry run isort . --check
+          poetry run ruff check
+          poetry run ruff format --check
```

Comment:
Using the following allows for ruff lints to be included in the PR files. I like it, maybe it's not consensual ?

```yml
        run: |
          poetry run ruff check --output-format github
```

Reply:
Didn't know about it and kind of nice. Unless someone else complains, Done in [f47f109f5a938ff5b9cc4e52cba43ad9d0b65fcf](https://github.com/OpenRailAssociation/osrd/compare/7fab4f32c5b6c6c33700fa1abb0495f754fa9fbb..f47f109f5a938ff5b9cc4e52cba43ad9d0b65fcf).